test/vagrant-service-manager/binary_handlers/adb_docker_binary_handler_test.rb

```diff
@@ -54,6 +54,11 @@ module ServiceManager
       end
 
       it 'should validate download url' do
+        @handler.build_download_url
+        assert_silent { @handler.validate_url }
```

Comment:
Didn't know about `assert_nothing_raise`. I was expecting to do it but found `assert_silent` somewhere while googling.